src/logic/wallets/getWeb3.ts

```diff
@@ -10,22 +10,21 @@ import { isValidCryptoDomainName } from 'src/logic/wallets/ethAddresses'
 import { getAddressFromUnstoppableDomain } from './utils/unstoppableDomains'
 
 // This providers have direct relation with name assigned in bnc-onboard configuration
-export const WALLET_PROVIDER = {
-  SAFE: 'SAFE',
-  METAMASK: 'METAMASK',
-  REMOTE: 'REMOTE',
-  TORUS: 'TORUS',
-  PORTIS: 'PORTIS',
-  FORTMATIC: 'FORTMATIC',
-  SQUARELINK: 'SQUARELINK',
-  WALLETCONNECT: 'WALLETCONNECT',
-  OPERA: 'OPERA',
+export enum WALLET_PROVIDER {
+  METAMASK = 'METAMASK',
+  TORUS = 'TORUS',
+  PORTIS = 'PORTIS',
+  FORTMATIC = 'FORTMATIC',
+  SQUARELINK = 'SQUARELINK',
+  WALLETCONNECT = 'WALLETCONNECT',
+  TRUST = 'TRUST',
+  OPERA = 'OPERA',
   // This is the provider for WALLET_LINK configuration on bnc-onboard
-  COINBASE_WALLET: 'COINBASE WALLET',
-  AUTHEREUM: 'AUTHEREUM',
-  LEDGER: 'LEDGER',
-  TREZOR: 'TREZOR',
-  LATTICE: 'LATTICE',
+  COINBASE_WALLET = 'COINBASE WALLET',
+  AUTHEREUM = 'AUTHEREUM',
+  LEDGER = 'LEDGER',
+  TREZOR = 'TREZOR',
+  LATTICE = 'LATTICE',
```

Comment:
The values can be removed.
```
export enum WALLET_PROVIDER {
   METAMASK,
   TORUS,
   ...
}
```

Reply:
The thing is we use it here https://github.com/gnosis/safe-react/blob/6efebedce448390f51704915b9284f4fe96fcdf8/src/logic/wallets/getWeb3.ts#L61 and that function expects a string to compare.

Reply:
Yes, that should still work. That's how enum works.